app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -0,0 +1,58 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.ProductNotFoundException;
+import com.codesoom.assignment.domain.Product;
+import com.codesoom.assignment.domain.ProductRepository;
+import com.codesoom.assignment.dto.request.ProductRequest;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+import java.util.List;
+
+@Service
+@Transactional
+public class ProductService {
+    private final ProductRepository productRepository;
+
+    public ProductService(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+    /**
+     *
+     * @return 모든 상품 목록을 리턴합니다.
+     */
+    public List<Product> findAll() {
+        return productRepository.findAll();
+
+    }
+
+    /**
+     * Client가 요청한 id에 해당하는 상품을 찾아 리턴합니다.
```

Comment:
좋습니다! 클라이언트는 굳이 언급하지 않아도 될 것 같아요.
```suggestion
     * 요청한 id에 해당하는 상품을 찾아 리턴합니다.
```